src/Service/ServiceUtils.cpp

```diff
@@ -280,149 +278,116 @@ static ErrorMessageOr<fs::path> FindSymbolsFilePathInStructuredDebugStore(
       absl::StrFormat("File does not exist: %s", path_in_structured_debug_store.string())};
 }
 
-static ErrorMessageOr<fs::path> FindSymbolsFilePathCoff(const fs::path& module_path,
-                                                        const CoffFile* module_coff_file) {
-  CHECK(module_coff_file != nullptr);
+ErrorMessageOr<fs::path> FindSymbolsFilePath(
+    const orbit_grpc_protos::GetDebugInfoFileRequest& request) {
```

Comment:
did you forget to modify GetDebugInfoFileRequest?

Reply:
No.... I am not sure what you mean.